assignments/array-methods.js

```diff
@@ -53,31 +53,77 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":49,"first_name":"Bel","last_name":"Alway","email":"balway1c@ow.ly","shirt_size":"S","company_name":"Voolia","donation":107},
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
+
+//runners
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+    runners.forEach(function (runner) {
```

Comment:
Great Job on these, trying going back through or thinking about how you can convert these to ES6 to make them easier to read. In this case you could do the following 
`let fullName = runner.forEach((runner) => `${runner.first_name} ${runner.last_name}``
some of the higher order functions return an array by default, check the mdn page to see which ones. If they do this then you don't need to initialize an empty array variable, you just toss the HOF into one and it will make it every time the variable is called.